src/Resources/doc/index.rst

```diff
@@ -0,0 +1,121 @@
+SymfonyMakerBundle
+==================
+
+Symfony Maker helps you creating empty commands, controllers, form classes,
+tests and more so you can forget about the required boilerplate code. This
+bundle is an alternative to `SensioGeneratorBundle`_ for modern Symfony
+applications and requires using Symfony 3.4 or newer and `Symfony Flex`_.
+
+Installation
+------------
+
+Run this command to install and enable this bundle in your application:
+
+.. code-block:: terminal
+
+    $ composer require maker
+
+Usage
+-----
+
+This bundle provides several commands under the ``maker:`` namespace. List them
+all executing this command:
+
+.. code-block:: terminal
+
+    $ php bin/console make:
```

Comment:
The command to run is `php bin/console list make`. Yours does not work AFAIK.

Reply:
It works ... but it's not pretty:

![command-1](https://user-images.githubusercontent.com/73419/32978361-238ccbce-cc41-11e7-9db6-17668719dde3.png)

Your proposal is much better (although the output is a bit noisy):

![command-2](https://user-images.githubusercontent.com/73419/32978364-2f489dda-cc41-11e7-9199-32a3cb741ec8.png)